### PR TITLE
used inheritance and callbacks.

### PR DESCRIPTION
**Pull request by hirenjethva**

### CODE REVIEW

File: Week-4/excersie1.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        setTimeout(() => console.log("Resolve the promise " + some),1000);
+    }
+}
+
+function getNumber(mycallback){
+    let randNum = Math.floor((Math.random() * 100) + 1);
+    mycallback(randNum);
+}
+
+getNumber(displayLog);
```
Comment (TeamMonuKumar) [inline Week-4/excersie1.js:38]: function getNumber(resolveTime, rejectTime){
    let randomNum = parseInt(Math.random()*100);
    console.log(randomNum);

    if (randomNum % 5 == 0){
        // Thenable object
        return {
            then(){
                setTimeout(() => console.log(`custom promise has been REJECTED after ${rejectTime}ms`), rejectTime)
            }
        }
    }
    return {
        // Thenable object
        then(){
            setTimeout(() => console.log(`custom promise has been RESOLVED after ${resolveTime}ms`), resolveTime)
        }
    }
}

getNumber(1000, 2000).then();

Comment (TeamMonuKumar) [reply Week-4/excersie1.js:38]: Write code in Thenable form

Comment (hirenjethva) [reply Week-4/excersie1.js:38]: Thanks monu for reviewing. Will update the recommendation method.

**Comment by hirenjethva** [comment]
as per conversation code has been modified.

**Pull request merged**